packages/repository/src/__tests__/integration/repositories/relation.factory.integration.ts

```diff
@@ -237,35 +237,25 @@ describe('HasManyThrough relation', () => {
   });
 
   it('creates a target instance alone with the corresponding through model', async () => {
-    const cartItem = await customerCartItemRepo.create(
-      {
-        description: 'an item hasManyThrough',
-      },
-      {
-        throughData: {id: 99},
-      },
-    );
```

Comment:
I like how the test suite is much simpler now.

However, do we have any tests to verify `throughData` behavior after this commit removed `throughData` from test cases where it's not explicitly needed/tested?

Reply:
Added a test to verify the specified `throughData`.